packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalk.js

```diff
@@ -70,9 +69,10 @@ function QuickTalk ({
 
   return (
     <QTPanel
+      a11yTitle='QuickTalk Comments Panel'
       elevation='medium'
+      role='dialog'
```

Comment:
Dialogs have some specific requirements so be careful, when adding the `dialog` role, that your component is a valid dialog. The panel might also need `aria-modal`. 
https://www.w3.org/TR/wai-aria-practices-1.2/

Reply:
You’ll also need to make sure that the dialog is focused after it opens. 
https://www.w3.org/TR/wai-aria-practices-1.2/#keyboard-interaction-7

Reply:
Noted. I did not realise using `role=dialog` would have a lot of additional functionality implications, but these will be addressed.

Reply:
To be honest, a popup needs to implement those even if it doesn't explicitly declare `role=dialog` in the accessibility tree. After it opens, the reading focus should be at the first comment (or the top of the dialog) and Tab/shift-Tab should navigate you forward and backward through the interactive elements in the comments box.